launcher/src/main/java/org/apache/spark/launcher/SparkAppHandle.java

```diff
@@ -19,7 +19,7 @@
 
 /**
  * A handle to a running Spark application.
- * <p/>
+ * <p>
```

Comment:
javadoc enforces a broken version of HTML where you don't need to close certain tags like `p`. It's a little annoying, but you still need the HTML to format the docs nicely.